### PR TITLE
Issue #148 Make it so that a set value of FALSE for autoPlay and/or playAll is reflected in the player URL

### LINKED ISSUE #148: Player Url class does not output autoPlay=false
Mpx players can be configured to autoPlay and playAll by default. Sometimes you want to override that behaviour on a per-player basis by setting `autoPlay=false` or `playAll=false` on the query string for the player URL. The current implementation ignores a value of FALSE.

### PR DESCRIPTION
**Pull request by m4olivei**
Resolves #148.

### CODE REVIEW

**Review by geekygnr — COMMENTED** [review]
No major issues visually.

File: src/Service/Player/Url.php [hunk 1]
```diff
@@ -109,11 +109,11 @@ public function __toString()
      *
      * @see https://docs.theplatform.com/help/player-player-autoplay
      *
-     * @param bool $autoplay True to enable autoplay, false otherwise.
+     * @param bool $autoPlay True to enable autoPlay, false otherwise.
      */
-    public function setAutoplay(bool $autoplay)
+    public function setAutoplay(bool $autoPlay)
```
Comment (geekygnr) [inline src/Service/Player/Url.php:114]: If we are following camel case should this be setAutoPlay similar to the variable autoPlay?

Comment (geekygnr) [inline src/Service/Player/Url.php]: If we are following camel case should this be setAutoPlay similar to the variable autoPlay?

Comment (m4olivei) [reply src/Service/Player/Url.php:114]: I hesitated to do that b/c that would change the public interface and be a breaking change. Plus require changing all the callers. I wanted to keep this low risk and quick.

Comment (geekygnr) [reply src/Service/Player/Url.php:114]: Sounds good to me

**Pull request merged**